docker-entrypoint.sh

```diff
@@ -23,7 +23,9 @@ fi
 if [ -d "/usr/local/tomcat/webapps/ors" ]; then
 	cp -f /ors-conf/app.config $tomcat_appconfig
```

Comment:
This copies the users app.config to `/usr/local/tomcat/webapps/ors/WEB-INF/classes/app.config` if `/usr/local/tomcat/webapps/ors` exists and then ignores it

Reply:
Correct, so here the config within the ors-conf folder is only copied to the tomcat webapp if openrouteservice has already been built and deployed (else the ors folder wont exist in webapps).If the ors folder doesnt exist (ors hasn't been built yet) then it would do the other option (see next comment)